src/constants/Constants.js

```diff
@@ -0,0 +1,12 @@
+const ACCESS_TOKEN = "accessToken";
```

Comment:
You have created these constants, why have not used these ?

Reply:
Actually, just forgot :')